prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir

```diff
@@ -1,8 +1,8 @@
 {
   "resourceType" : "Bundle",
-  "id" : "64072005-fd2d-49cf-a3e8-a18567b46938",
```

Comment:
I've fully read through the entire file and can confirm that the only changes are the ones that I added and the ids + references to them.